src/main/java/org/infinispan/persistence/cloud/configuration/Attribute.java

```diff
@@ -7,27 +7,35 @@
  * Enumerates the attributes used by the Cloud cache store configuration
  *
  * @author Tristan Tarrant
- * @since 5.2
+ * @author Damiano Albani
+ * @since 6.0
  */
 public enum Attribute {
    // must be first
    UNKNOWN(null),
 
-   BUCKET_PREFIX("bucketPrefix"),
-   CLOUD_SERVICE("cloudService"),
-   CLOUD_SERVICE_LOCATION("cloudServiceLocation"),
-   COMPRESS("compress"),
+   PROVIDER("provider"),
    IDENTITY("identity"),
-   LAZY_PURGING_ONLY("lazyPurgingOnly"),
-   MAX_CONNECTIONS("maxConnections"),
-   PASSWORD("password"),
-   PROXY_HOST("proxyHost"),
-   PROXY_PORT("proxyPort"),
-   REQUEST_TIMEOUT("requestTimeout"),
-   SECURE("secure"),
-
+   CREDENTIAL("credential"),
+   CONTAINER("container"),
+   KEY_TO_STRING_MAPPER("key2StringMapper"),
```

Comment:
Can you please follow same convention used as for JDBC cache store? It's

    KEY_TO_STRING_MAPPER("key-to-string-mapper")